modules/virt-importing-vm-cli.adoc

```diff
@@ -81,6 +81,14 @@ EOF
 <5> Specify `nfs` or `ocs-storagecluster-ceph-rbd`.
 <6> If you specified the `ocs-storagecluster-ceph-rbd` storage class, you must specify `Block` as the volume mode.
 
+// For CNV 2.6.1 only. Will be fixed in CNV 2.6.2.
```

Comment:
Because of this comment (I believe), the numbering is starting over at 1 with the next bullet.

Reply:
Thanks! I must have forgotten to check the build here. Hit me over the head with an IBM style guide.... :-)